features/claims/advocate/scheme_sixteen/advocate_supplementary_claim_submit.feature

```diff
@@ -0,0 +1,154 @@
+@javascript
+Feature: Advocate tries to submit a fee scheme 16 supplementary claim for miscellaneous fees (only)
+
+  @fee_calc_vcr
+  Scenario: I create a supplementary claim but skip adding only applicable misc fees
+
+    Given the current date is '2023-11-16'
+    And I am a signed in advocate
+    And I am on the 'Your claims' page
+    And I click 'Start a claim'
+    And I select the fee scheme 'Advocate supplementary fee'
+
+    Then I should be on the advocate supplementary new claim page
+
+    When I enter a providers reference of 'AGFS supplementary fee test'
+    And I select the court 'Caernarfon'
+    And I enter a case number of 'A20191234'
+    And I enter scheme 16 main hearing date
+
+    And I should see a page title "Enter case details for advocate supplementary fee claim"
+    Then I click "Continue" in the claim form
+
+    And I enter defendant, scheme 16 representation order and MAAT reference
+    And I add another defendant, scheme 16 representation order and MAAT reference
+
+    Given I insert the VCR cassette 'features/claims/advocate/scheme_sixteen/supplementary_fee_calculations'
```

Comment:
curious, does this exist? i can't find it

Reply:
The VCR files should all be in the PR now :-) @Obsiye